charts/fluid/fluid/CHANGELOG.md

```diff
@@ -54,3 +54,4 @@
 
 ### 0.9.0
 * Support pass image pull secrets from fluid charts to alluxioruntime controller
+* Dataload support Affinity,Tolerations,NodeSelector,SchedulerName
```

Comment:
Please instead update CHANGELOG.md of the fluid-dataloader chart

Reply:
Thanks for the reminder.